ipatests/test_integration/test_installation.py

```diff
@@ -66,6 +68,66 @@ def wrapped(*args):
     return wrapped
 
 
+def assert_mod(actual_mod, expected_mod):
+    """
+    Compare the given permission sets
+    """
+    assert oct(actual_mod) == oct(expected_mod)
+
+
+def assert_in_mod(mod, expected_bit):
+    """
+    Check whether an expected permission bit is within the given set
+    """
+    if not mod & expected_bit:
+        raise AssertionError(
+            "Permission set {0} doesn't contain expected {1}".format(
+                oct(stat.S_IMODE(mod)), "{0:#06o}".format(expected_bit)
+            )
+        )
+
+
+def assert_path(path, username):
+    """
+    Check whether the given user owns the given path
+    """
+    print("check for {}".format(path))
```

Comment:
Did you mean to leave the print statements?

Reply:
`assert_path` is called recursively on a full path. I think this should help to debug failures if any. But I do not insist :)